docs/content/docs/scale.md

```diff
@@ -0,0 +1,110 @@
+		- 2 replicaset per server service
+	- 1 TrafficSplitt between the 5 server services (10 pods backed)
+	- Total of 20 pods each iteration. 10 client pods will REST GET the Traffic split.
+	- Correctness is ensured. It is checked that all TrafficSplitt server members are eventually reached.
+- Test timeout for network correctness: 150 seconds
+
+#### Varying form factors
+- with 256MB max RAM
+	- 1.5 CPU
+- With 512MB RAM
+	- 1.5 CPU
+	- 4 CPU
+
+(All tests were run multiple times to ensure validity of the results)
+
+## Assessment
+
+Note: Please assume we are deploying a proxy per pod in the following results, so use pod/proxies interchangeably.
+### Memory
+- Memory per pod/envoy onboarded in the network is calculated after the initial snapshot with nothing onboarded on the mesh is seen to take into account standalone memory used by OSM.
+  - Initial memory use of OSM **40~46MB**
+  - Memory increase per pod ranges between **800KB~1MB** per pod. 
+- With **256MB** max memory limit for OSM's container, **180-200 pods** should be a conservative limit. Any more pods
+can (and will) have OSM `OOMKilled` by Kubernetes.
+- With **512MB** max memory limit for OSM's container, it can reach to **~400 pods**, although convergence times will spike through the roof up to **+120s** with a rather oversubscribed CPU profile, that could potentially lead to other issues.
+
+### CPU
+#### 1.5/2 vCPU (512MB mem)
+- At **200 pods**, ADS completion times<sup>[1]</sup> for individual envoys range around **10s-20s** and full configuration convergence ranges around **20-40s**
+- At **400 pods**, ADS completion times<sup>[1]</sup> for individual envoys range around **40-90s** and full configuration convergence ranges around **+150s**
+- Injector webhook and certificate (2048 RSA) issuance times are stable between **1s-5s** in all cases.
+#### 4 vCPU (512MB mem)
+- At **200 pods**, ADS completion times<sup>[1]</sup> for individual envoys range around **5-10s**, however full configuration convergence falls into the **20-40s** bucket.
+- At **400 pods**, ADS completion times<sup>[1]</sup> for individual envoys range around **20-40s** and full configuration convergence ranges around **+90s**
+- Injector webhook and certificate (2048 RSA) issuance times are stable between **1s-5s** in all cases.
+
+- In both cases, and taking into account the architecture limitations explained above, OSM will run hot on CPU, most of the time recomputing ADS updates for all pods in the mesh. This oversubscription could lead to not enough cycles available for other critical path routines (liveness), depending entirely on the scheduler.
+- Adding more vCPU helps OSM compute a larger amount of ADS updates at a time, as well as reduce the configuratin convergence time - however due to the nature of the buckets, we suspect this is far from a linear improvement as some of these numbers could suggest.
+
+
+Below, more results with meaningful related data explained.
+
+## Detailed Test Results
```

Comment:
There's awesome data here. I think it would be helpful to add a section to help generally answer the question, "If I need to run X number of pods in the mesh and keep ADS times under Y seconds, how much memory and CPU do I need for the osm-controller?"

I understand if that's not really the goal of this doc, but if there's a somewhat straightforward way to answer that I think it might be valuable.

Reply:
> If I need to run X number of pods in the mesh and keep ADS times under Y seconds, how much memory and CPU do I need for the osm-controller?
That's actually a great idea. 
Some of the metric given attempt to give some insight on those (like, mem/pod), but for cpu is probably more complicated since the cpu costs are nowhere linear per pod or per ads update time...